### ISSUE TITLE
Could we get data about number of moderator vs admin removals?

### ISSUE THREAD

**Issue by go1dfish** [drunken-economist/reddit-10-year-data#1]
You know for transparency and all that.

**Comment by lutze** [comment]
That could be interesting to see.

**Comment by ghost** [comment]
Yeah, I'll run it now. Admin removal is easy, mod removal is harder (I need to figure out the difference between a removal and a spam filter)

**Comment by ghost** [comment]
Over ten years, 22,724 were admin-removed, and 27,345 were admin-approved. I'm going to dig to figure out the spam filter vs mod-removed

**Comment by go1dfish** [comment]
Cool thanks, was hoping for a by/date breakdown alongside https://github.com/drunken-economist/reddit-10-year-data/blob/master/thingsByMonth.csv to see trends if at all possible.
